tests/RouterTest.php

```diff
@@ -60,6 +63,38 @@ public function testAddClearRoute()
         $this->assertCount(0, $this->router->getRoutes());
     }
 
+    public function testAddRouteWithName()
+    {
+        $route = new Route('/foo/bar', [], ['controller' => '[a-z-]+', 'action' => '[a-z-]+']);
+
+        $this->assertCount(3, $this->router->getRoutes());
```

Comment:
This test can be easily broken by changes in other parts of this test. What do you think about the following approach instead:
```
$numberOfRoutes = count($this->router->getRoutes());
$this->router->addRoute($route, 'foo-bar');
$numberOfRoutes++;
$this->assertCount($numberOfRoutes, ($routes = $this->router->getRoutes()));
```

(same for `testAddRouteWithHttpMethods()`)